<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1172</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1172</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home Allenatore</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>725</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1172</width>
     <height>725</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>homeAllenatore.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestSchede">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;

</string>
   </property>
   <property name="text">
    <string>GESTIONE SCHEDE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestIscritti">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;

</string>
   </property>
   <property name="text">
    <string>GESTIONE ISCRITTI CORSO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_visualizzaCliente">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>360</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;

</string>
   </property>
   <property name="text">
    <string>VISUALIZZA CLIENTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_visualizzaTurni">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;

</string>
   </property>
   <property name="text">
    <string>VISUALIZZA TURNI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>590</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;

</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Allenatore.qrc"/>
 </resources>
 <connections/>
</ui>
